.travis.yml

```diff
@@ -41,8 +41,8 @@ matrix:
   - php: "5.2"
     env: WP_MODE=multi  WP_BRANCH=master
     dist: precise
-  - env: WP_TRAVISCI="yarn lint"
-  - env: WP_TRAVISCI="yarn test-dangerci-and-adminpage"
+  - language: node_js
```

Comment:
Using `language:node_js` will let Travis pick us a Node.js container based on the version we have in `.nvmrc`.